lib/event.rb

```diff
@@ -0,0 +1,27 @@
+
+class Event
+  attr_reader :ages, :name
+  def initialize(name, ages)
+    @ages = ages
+    @name = name
+  end
+
+  def max_age
+    ages.max
+  end
+
+  def min_age
+    ages.min
+  end
+
+  def average_age
+    ages_sum = ages.sum.round(1)
```

Comment:
I assume you are doing the round(1) in order to convert to a float? If so, you can instead do `mean_age = (ages_sum.to_f / ages.length) on the line below. Not a huge issue though.